### PR TITLE
Composer and override fixes

### PR DESCRIPTION
**Pull request by jozefspisiak**
I am not really sure if this is breaking, but I marked it anyway, fixed the namespace prefix, so it works with the new added composer file.

For the migration of TCA, it is mainly to allow easy overrides with other extensions. I would like to get review from both of you before we merge this. Thanks

### CODE REVIEW

File: Classes/Controller/CategoriesController.php [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <?php
-namespace PXA\PxaDealers\Controller;
+namespace Pixelant\PxaDealers\Controller;
 
 use \TYPO3\CMS\Extbase\Utility\DebuggerUtility as du;
```
Comment (MattiasNilsson) [inline Classes/Controller/CategoriesController.php]: Remove the DebuggerUtility use statement

File: Classes/Controller/ClosestController.php [hunk 2]
```diff
@@ -1,5 +1,5 @@
 <?php
-namespace PXA\PxaDealers\Controller;
+namespace Pixelant\PxaDealers\Controller;
 
 use \TYPO3\CMS\Extbase\Utility\DebuggerUtility as du;
```
Comment (MattiasNilsson) [inline Classes/Controller/ClosestController.php]: Remove the DebuggerUtility use statement

File: Classes/Controller/CountriesController.php [hunk 3]
```diff
@@ -1,5 +1,5 @@
 <?php
-namespace PXA\PxaDealers\Controller;
+namespace Pixelant\PxaDealers\Controller;
 
 use \TYPO3\CMS\Extbase\Utility\LocalizationUtility as lu;
```
Comment (MattiasNilsson) [inline Classes/Controller/CountriesController.php]: Remove the DebuggerUtility use statement

File: Classes/Controller/DealersController.php [hunk 4]
```diff
@@ -1,5 +1,5 @@
 <?php
-namespace PXA\PxaDealers\Controller;
+namespace Pixelant\PxaDealers\Controller;
 
 use TYPO3\CMS\Extbase\Utility\DebuggerUtility;
```
Comment (MattiasNilsson) [inline Classes/Controller/DealersController.php]: Remove the DebuggerUtility use statement

File: Classes/Domain/Model/Dealers.php [hunk 5]
```diff
@@ -1,5 +1,5 @@
 <?php
-namespace PXA\PxaDealers\Domain\Model;
+namespace Pixelant\PxaDealers\Domain\Model;
 
 use \TYPO3\CMS\Extbase\Utility\DebuggerUtility as du;
```
Comment (MattiasNilsson) [inline Classes/Domain/Model/Dealers.php]: Remove the DebuggerUtility use statement

File: Classes/Domain/Repository/CountriesRepository.php [hunk 6]
```diff
@@ -1,5 +1,5 @@
 <?php
-namespace PXA\PxaDealers\Domain\Repository;
+namespace Pixelant\PxaDealers\Domain\Repository;
 
 use \TYPO3\CMS\Extbase\Utility\DebuggerUtility as du;
```
Comment (MattiasNilsson) [inline Classes/Domain/Repository/CountriesRepository.php]: Remove the DebuggerUtility use statement

File: Classes/Domain/Repository/DealersRepository.php [hunk 7]
```diff
@@ -1,5 +1,5 @@
 <?php
-namespace PXA\PxaDealers\Domain\Repository;
+namespace Pixelant\PxaDealers\Domain\Repository;
 
 use FluidTYPO3\Flux\Utility\DebuggerUtility;
```
Comment (MattiasNilsson) [inline Classes/Domain/Repository/DealersRepository.php]: Remove the DebuggerUtility use statement

**Comment by MattiasNilsson** [comment]
This looks good just some smaller nitpicks. But when it comes to breaking or not I still agree that it is breaking. The plugin gets new namespace. 👍

**Comment by jozefspisiak** [comment]
Removed the debugger statements now and yes, you are right. Luckily as far as I know all plugins are used only on like 2 or 3 pages, so we can fix those quite easily.

**Comment by MattiasNilsson** [comment]
Looks good now! 👍

**Comment by pixelmatseriks** [comment]
I agree, looks good.

**Comment by vuxorpadd** [comment]
Same here. Looks good!

**Comment by MattiasNilsson** [comment]
Then we merge! 👍

**Pull request merged**